tasks/main.yml

```
@@ -627,6 +627,193 @@
         __mssql_conf_setting: "network forceencryption"
         __mssql_conf_setting_value: "{{ '1' if mssql_tls_enable else 'unset' }}"
 
+- name: Configure Active Directory authentication
+  vars:
+    __mssql_kinit_user: >-
+      {{ ad_integration_user }}@{{ ad_integration_realm
+      | upper }}
+  when: mssql_ad_configure | bool
+  block:
+    - name: Join to {{ ad_integration_realm }} realm
+      vars:
+        ad_integration_realm: "{{ ad_integration_realm }}"
+        ad_integration_password: "{{ ad_integration_password }}"
+      include_role:
+        name: linux-system-roles.ad_integration
+
+    - name: Ensure the adutil package
+      package:
+        name: adutil
+        state: present
+
+    - name: List valid kerberos tickets
+      shell: klist || true
```

```suggestion
      command: klist
```
Since you're using `ignore_errors`
Also, you might want to use `klist -A` in case there are multiple principals on the machine.